client/lib/gsuite/get-google-mail-service-name.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal Dependencies
+ */
+import config from 'calypso/config';
+import { GSUITE_PRODUCT_NAME, GOOGLE_WORKSPACE_PRODUCT_NAME } from 'calypso/lib/gsuite/constants';
+
+export function getGoogleMailServiceName( productName = null ) {
```

Comment:
I have the feeling this function should accept a product identifier rather than a product name but maybe I'm wrong.